client-src/elements/chromedash-link.js

```diff
@@ -109,6 +111,45 @@ function enhanceGithubIssueLink(featureLink, text) {
   const typePath = featureLink.url.split('/').slice(-2)[0];
   const type = typePath === 'issues' ? 'Issue' : typePath === 'pull' ? 'Pull Request' : typePath;
 
+  // If this issue is an external review of the feature, find the summary description.
+  const externalReviewer = ExternalReviewer.get(repo);
+  let stateDescription = undefined;
+  let stateVariant = undefined;
+  if (externalReviewer) {
+    for (const label of information.labels) {
+      const labelInfo = externalReviewer.label(label);
+      if (labelInfo) {
+        ({description: stateDescription, variant: stateVariant} = labelInfo);
+        break;
+      }
+    }
+  }
+
+  if (stateVariant === undefined) {
+    if (state === 'open') {
+      const age = Date.now() - createdAt.getTime();
+      stateDescription = html`Opened <relative-time .date=${createdAt} format="relative" precision="day" threshold="P100Y">on ${_dateTimeFormat.format(createdAt)}</relative-time>`;
```

Comment:
We already have `<sl-relative-time>` as used in utils.js.  Could you use that? It might even just be wrapping the same underlying library.

Reply:
`<sl-relative-date>` is missing the `precision` argument that I'm using to turn "Opened 2 Hours Ago" into "Opened Today" (once https://github.com/github/relative-time-element/pull/280 goes in), but that's not a big deal if you'd rather avoid the dependency. I'll see if there are any other problems once my development machine finishes upgrading.

Reply:
I have to undo the style from https://github.com/GoogleChrome/chromium-dashboard/blob/cffa57147e486fc1e44e972740306c461e884342/client-src/css/shared-css.js#L191-L193
but then it seems to work:
![image](https://github.com/GoogleChrome/chromium-dashboard/assets/83420/e4b3e9dd-98f8-4068-a1aa-b4dcf11b63f6)

Here's a patch without `<relative-time>`.